odalic/src/main/java/cz/cuni/mff/xrg/odalic/outputs/rdfexport/DefaultAnnotatedTableToRDFExportAdapter.java

```diff
@@ -0,0 +1,136 @@
+package cz.cuni.mff.xrg.odalic.outputs.rdfexport;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Value;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.model.impl.LinkedHashModel;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.model.vocabulary.DCTERMS;
+import org.eclipse.rdf4j.model.vocabulary.RDF;
+
+import cz.cuni.mff.xrg.odalic.input.Input;
+import cz.cuni.mff.xrg.odalic.outputs.annotatedtable.AnnotatedTable;
+import cz.cuni.mff.xrg.odalic.outputs.annotatedtable.TableColumn;
+
+/**
+ * The default {@link AnnotatedTableToRDFExportAdapter} implementation.
+ * 
+ * @author Josef Janoušek
+ *
+ */
+public class DefaultAnnotatedTableToRDFExportAdapter implements AnnotatedTableToRDFExportAdapter {
+
+  /**
+   * The default toRDFExport implementation.
+   * 
+   * @see cz.cuni.mff.xrg.odalic.outputs.rdfexport.AnnotatedTableToRDFExportAdapter#toRDFExport(cz.cuni.mff.xrg.odalic.outputs.annotatedtable.AnnotatedTable, cz.cuni.mff.xrg.odalic.input.Input)
+   */
+  @Override
+  public Model toRDFExport(AnnotatedTable annotatedTable, Input extendedInput) {
+    
+    ValueFactory factory = SimpleValueFactory.getInstance();
+    
+    // create a new Model to put statements in
+    Model model = new LinkedHashModel();
+    
+    HashMap<String, Integer> columnsNamesPositionsInput = new HashMap<>();
+    int i = 0;
+    for (String headerName : extendedInput.headers()) {
+      columnsNamesPositionsInput.put(headerName, i);
+      i++;
+    }
+    
+    HashMap<String, TableColumn> namesColumnsAnnotatedTable = new HashMap<>();
+    for (TableColumn column : annotatedTable.getTableSchema().getColumns()) {
+      namesColumnsAnnotatedTable.put(column.getName(), column);
+    }
+    
+    // fetch the relations in appropriate order
+    HashMap<String, HashMap<String, HashMap<String, String>>> relations = new HashMap<>();
```

Comment:
This is awful, why didn't you represent Relation as a class, which has certain URL, subject and object, right?